tools/build_variables.bzl

```
@@ -562,6 +562,12 @@ libtorch_python_distributed_sources = [
     "torch/csrc/jit/runtime/register_distributed_ops.cpp",
 ]
 
+libtorch_python_distributed_win32_sources = [
```

To avoid duplication, please rename this list `python_distributed_core_sources` and make `distributed_source` sources an add-on to that one? (Or just call it distributed_extra sources and append it for non-windows platforms)
```suggestion
libtorch_python_distributed_core_sources = [
```